tests/cpp/unit-tests/test_client.cpp

```diff
@@ -414,3 +423,34 @@ SCENARIO("Testing Client Object", "[Client]")
         }
     }
 }
+
+SCENARIO("Testing INFO Functions on Client Object", "[Client]")
+{
+
+    GIVEN("A Client object")
+    {
+        Client client(true);
+
+        THEN("Calling INFO on database or cluster with "
+             "an invalid address throws errors")
+        {
+            std::string db_address = ":00";
+
+            CHECK_THROWS_AS(
+                client.get_db_node_info(db_address),
+                std::runtime_error);
+            CHECK_THROWS_AS(
+                client.get_db_cluster_info(db_address),
+                std::runtime_error);
+        }
+
+        AND_THEN("Calling INFO on database or cluster with "
+                "a valid address does not throw errors")
+        {
+            std::string db_address = "127.0.0.1:6379";
```

Comment:
This would only be true for a locally launched cluster / non-cluster.  I think the test suite has a method to read in the ``SSDB`` variable.